src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -43,6 +43,12 @@ def get_attrs(self, item_list, user, **kwargs):
             rule_result = result[alert_rules[alert_rule_id]].setdefault("projects", [])
             rule_result.append(project_slug)
 
+        triggers = AlertRuleTrigger.objects.filter(alert_rule__in=item_list).order_by("label")
+        serialized_triggers = serialize(list(triggers))
+        for trigger in serialized_triggers:
+            alert_rule_triggers = result[alert_rules[alert_rule_id]].setdefault("triggers", [])
```

Comment:
Where is `alert_rule_id` coming from? It looks like you're getting the last value from the loop above, and that this will append all the triggers to one alert rule when there could be multiple results.

Reply:
Damn, good catch. Looks like the tests missed this since we only have one alert rule.